doc/en/user/source/extensions/css/workshop/point.rst

```diff
@@ -0,0 +1,708 @@
+.. _extensions_css_cartography_point:
+
+Points
+======
+
+The next stop of the CSS styling tour is the representation of points. 
+
+.. image:: img/PointSymbology.svg
+
+Review of point symbology:
+
+* Points are used to represent a location only, and do not form a shape. The visual width of lines do not change depending on scale.
+
+* SLD uses a **PointSymbolizer** record how the shape of a line is drawn.
+
+* Labeling of points is anchored to the point location.
+
+As points have no inherent shape of of their own, emphasis is placed on marking locations with an appropriate symbol.
+
+Reference:
+
+* `Point Symbology <../properties.html#point-symbology>`_ (User Manual | CSS Property Listing)
+* `Points <../cookbook/point.html>`_ (User Manual | CSS Cookbook)
+* `Styled Marks <../styledmarks.html>`_ (User Manual | CSS Styling )
+* `Point <../../../styling/sld-reference/pointsymbolizer.html>`_ (User Manual | SLD Reference )
+
+This exercise makes use of the ``ne:populated_places`` layer.
+
+#. Navigate to the **CSS Styles** page.
+
+#. Click :guilabel:`Choose a different layer` and select :kbd:`ne:populated_places` from the list.
+
+   .. image:: img/point_01_preview.png
```

Comment:
Don't know how much we care, but this is a screenshot including layers that users won't have. Maybe a judicious crop?